ruby/vampires.rb

```diff
@@ -0,0 +1,77 @@
+attempts = 0
+
+until attempts == 4
+	attempts += 1
+
+puts "What is your name?"
+name = gets.chomp
+	
+
+def age_finder(x, y)
+		 (2016 - y.to_i) == x.to_i
```

Comment:
Check out the Ruby Time class http://ruby-doc.org/core-2.2.0/Time.html that you could use instead of hardcoding the year to `2016`. There's a pretty easy way to extract the year